Gothic-Unity-Project/Gothic-Unity
# Feature/50 add quest log first iteration

## To test
* Start the game with VRDeviceSimulator
* Hit "L"
* Check if you can click different options in the Quest Log
* Does everything work as expected?

Hints:
* I highly recommend you use one of these save games as they contain a lot of data: https://discord.com/channels/1246501201135010045/1304051091431882782/1305184341369950309

Out-of-scope:
* I didn't implement a proper mechanism to open the menu via normal VR gameplay (button etc.) I add this to a second ticket (#177).

Impressions:

![image](https://github.com/user-attachments/assets/e2706577-eb23-463c-a4ac-f2302d323d5a)
---
![image](https://github.com/user-attachments/assets/0712bd63-8305-4f4a-9c8e-5f5d89d691c9)
